driver/src/main/scala/api/database.scala

```diff
@@ -127,27 +127,8 @@ trait DBMetaCommands {
   /** Returns an index manager for this database. */
   def indexesManager(implicit ec: ExecutionContext) = new IndexesManager(self)
 
-  private lazy val collectionNameReader =
```

Comment:
@sgodbillon Let me know what you think about moving code in this way, to `DefaultDB` (to be able to use `runCommand` with `{ 'listCollections': 1 }`).